Shared Frameworks/HackersKit/HtmlParser.swift

```diff
@@ -42,8 +42,8 @@ enum HtmlParser {
         guard let id = Int(try postElement.attr("id")) else {
             throw Exception.Error(type: .SelectorParseException, Message: "Couldn't parse post ID")
         }
-        let urlString = try postElement.select(".titlelink").attr("href")
-        let title = try postElement.select(".titlelink").text()
+        let urlString = try postElement.select(".titleline").select("a").attr("href")
+        let title = try postElement.select(".titleline").text()
```

Comment:
This has resulted in the URL being included in the title as displayed in the app along with the link also being part of the article component. Should this line be instead?
```swift
let title = try postElement.select(".titleline").select("a").text()
```

Reply:
Yep you're right, thanks for spotting this.